packages/paypal-js/package.json

```
@@ -3,11 +3,22 @@
     "version": "8.2.0",
     "description": "Loading wrapper and TypeScript types for the PayPal JS SDK",
     "main": "index.js",
-    "module": "dist/esm/paypal-js.js",
+    "module": "dist/v5/esm/paypal-js.js",
     "types": "types/index.d.ts",
     "publishConfig": {
         "access": "public"
     },
+    "exports": {
+        ".": {
+            "types": "./types/index.d.ts",
+            "import": "./dist/v5/esm/paypal-js.js",
+            "require": "./dist/v5/cjs/paypal-js.js"
+        },
+        "./v6": {
```

Thoughts on renaming this so its clear that its referring to the v6 web sdk?
```suggestion
        "./sdk-v6": {
```